doc/FAQ.rst

```diff
@@ -197,26 +270,44 @@ consistent across HoloViews backends, but in general HoloViews tries
 to use each backend's defaults where possible.
 
 
-**Q: Help! I don't know how to index into my object!**
+**Q: Why do my HoloViews and GeoViews objects work fine separately but
+are mismatched when overlaid?
 
-**A:**  In any Python session, you can look at ``print(obj)`` to see
-the structure of ``obj``. For
-an explanation of how this information helps you index into your
-object, see our `Composing Elements <user_guides/Composing_Elements.html>`_
-user guide.
+**A:** GeoViews works precisely the same as HoloViews, except that
+GeoViews is aware of geographic projections.  If you take an
+``hv.Points()`` object in lon,lat coordinates and overlay it on a
+GeoViews map in Web Mercator, the HoloViews object will be in
+entirely the wrong coordinate system, with the HoloViews object all
+appearing at one tiny spot on the globe.  If you declare the same
+object as ``gv.Points``, then GeoViews will (a) assume it is in
+lon,lat coordinates (which HoloViews cannot assume, as it knows
+nothing of geography), and (b) convert it into the coordinates
+needed for display (e.g. Web Mercator).  So, just make sure that
+anything with geographic coordinates is defined as a GeoViews object,
+and make sure to declare the coordinates (``crs=...``) if the data is
+in anything other than lon,lat.
 
 
-**Q: Help! How do I find out the options for customizing the
-appearance of my object?**
+**Q: Where have my custom styles gone after unpickling my object?**
 
-**A:** If you are in the IPython/Jupyter Notebook you can use the cell magic
-``%%output info=True`` at the top of your code cell. This will
-present the available style and plotting options for that object.
+**A:** HoloViews objects are designed to pickle and unpickle your core
+data only, if you use Python's ``pickle.load`` and
+``pickle.dump``. Because custom options are kept separate from
+your data, you need to use the corresponding methods ``Store.dump`` and
+``Store.load`` if you also want to save and restore per-object
+customization. You can import ``Store`` from the main namespace with
+``from holoviews import Store``.
 
-The same information is also available in any Python session using
-``hv.help(obj)``. For more information on customizing the display
-of an object, see our `Customizing Plots <user_guides/Customizing_Plots.html>`_
-user guide.
+
+**Q: Why are the sizing options so different between the Matplotlib
+and Bokeh backends?"**
+
+**"A:** The way plot sizes are computed is handled in radically
+different ways by these backends, with Matplotlib building plots 'inside
+out' (from plot components with their own sizes) and Bokeh building
+them 'outside in' (fitting plot components into a given overall size).
+Thus there is not currently any way to specify sizes in a way that is
+comparable between the two backends.
```

Comment:
This bit seems like it should move up to be merged with the earlier question about how to size plots in mpl and bokeh.

Reply:
maybe keep this PR for just reorganization and do this in another PR?

Reply:
Sounds good!